monetio/models/raqms.py

```diff
@@ -9,13 +9,15 @@
 import xarray as xr
 
 
-def open_dataset(fname, *, surf_only=False):
+def open_dataset(fname, convert_to_ppb=True, surf_only=False):
```

Comment:
```suggestion
def open_dataset(fname, *, convert_to_ppb=True, surf_only=False):
```
I think it makes sense for everything except `fname` to be keyword-only